src/commands/deployment/onboard.ts

```diff
@@ -35,6 +36,7 @@ export const onboardCommandDecorator = (command: commander.Command): void => {
     )
     .action(async opts => {
       try {
+        await addVariableGroupWithKeyVaultMap();
```

Comment:
should we make this part optional, or add a command line argument, in case they don't want to create a variable group when onboarding? Right now, it seems like this is a mandatory step within `onboard`

Reply:
It was only for testing and will removed in the final commit.